src/test/java/de/retest/ui/PathTest.java

```diff
@@ -52,4 +52,10 @@ public void test_parentPath() {
 		assertThat( path1.isParent( path1 ) );
 		assertThat( !Path.path( ELEMENT_1 ).isParent( path1 ) );
 	}
+
+	@Test
+	public void trailing_slash_should_not_cause_exception() {
+		Path.fromString( "/html/body/div/" );
```

Comment:
Isn't this an invalid XPath?

Reply:
That is valid xpath. It's not what we produce. but valid none-the-less...